tests/Yaapii.Atoms.Tests/Text/StartsWithTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Xunit;
+using Yaapii.Atoms.Text;
+
+namespace Yaapii.Atoms.Tests.Text
+{
+    public sealed class StartsWithTests
+    {
+        [Fact]
+        public void FindsTextStart()
+        {
+            var x =
+                new StartsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("Im a")
+                );
+            Assert.True(x.Value());
+        }
+
+        [Fact]
+        public void TakesStartAsString()
+        {
+            var x =
+                new StartsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    "Im a"
+                );
+            Assert.True(x.Value());
+        }
+
+        [Fact]
+        public void FailsOnWrongStart()
+        {
+            var x =
+                new StartsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("m an")
+                );
+            Assert.False(x.Value());
+        }
+
+        [Fact]
+        public void FailsOnCaseMismatch()
+        {
+            var x =
```

Comment:
same as above, not needed

Reply:
@Meerownymous
This test checks if case sensitive is activated or not in the implementing class. In my opinion the test is necessary.

Reply:
see above. These tests are simply too much.